openslides_backend/action/util/typing.py

```diff
@@ -20,14 +18,3 @@
 )
 
 ActionError = Any
-
-
-class Action(Protocol):  # pragma: no cover
```

Comment:
You have to re-add this protocol (and also rename it). It is used in http/views.py.

Reply:
I don't see why it's needed there, it was only used for one cast